lib/Backend/IREE/LowerLinkage.cpp

```diff
@@ -16,10 +16,20 @@ using namespace mlir::NPCOMP;
 using namespace mlir::NPCOMP::IREEBackend;
 
 namespace {
-#define GEN_PASS_REGISTRATION
-#include "npcomp/Backend/IREE/Passes.h.inc"
-} // end namespace
+// This pass lowers the public ABI of the module to the primitives exposed by
+// the refbackrt dialect.
+class LowerLinkagePass : public LowerLinkageBase<LowerLinkagePass> {
+  void runOnOperation() override {
+    ModuleOp module = getOperation();
+    for (auto func : module.getOps<FuncOp>()) {
+      if (func.getVisibility() == SymbolTable::Visibility::Public)
+        func->setAttr("iree.module.export", UnitAttr::get(&getContext()));
```

Comment:
I think we don't need this anymore

Reply:
Awesome. :)

Reply:
Removed in https://github.com/llvm/mlir-npcomp/pull/273